code/data_structures/src/stack/Stack using Linked-Lists.cpp

```diff
@@ -0,0 +1,75 @@
+#include<iostream>
+using namespace std;
+
+template <typename T>
+
+class Node {
+    public :
```

Comment:
Remove space before `:` and align with class level.

```C++
class Node {
public:
    // ....
};
```

On the other hand, just use a `struct` and then you don't have to deal with access modifiers, since all members are public anyway.